packages/desktop-client/src/components/sidebar/Account.tsx

```diff
@@ -148,6 +148,12 @@ export function Account({
             </View>
 
             <AlignedText
+              style={
+                (name === 'Off budget' || name === 'For budget') && {
+                  borderBottom: `1.5px solid rgba(255,255,255,0.4)`,
+                  paddingBottom: '3px;',
```

Comment:
After updating some branches with master, there is a console log all the time:

![image](https://github.com/actualbudget/actual/assets/15043428/c42ec15d-248a-492b-9671-33ced88bc0fb)

Reply:
Ah crap sorry... Didn't know about this. I'll create a PR tonight to fix this.

Reply:
#2881